plugin/storage/cassandra/schema/v001.cql.tmpl

```diff
@@ -188,14 +188,13 @@ CREATE TYPE IF NOT EXISTS ${keyspace}.dependency (
 );
 
 -- compaction strategy is intentionally different as compared to other tables due to the size of dependencies data
--- note we have to write ts twice (once as ts_index). This is because we cannot make a SASI index on the primary key
 CREATE TABLE IF NOT EXISTS ${keyspace}.dependencies (
-    ts          timestamp,
-    ts_index    timestamp,
+    ts           timestamp,
+    date_bucket  text
```

Comment:
shouldn't this be `int`?

Reply:
I initially had buckets in a different format, this can be `int`, or `bigint` if we anticipate smaller buckets in the future.